src/objects/feature-group/feature-group.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import defaultDemo from './demo/demo.twig';
+
+<Meta title="Objects/Feature Group" />
+
+# Feature Group
+
+The Feature Group layout object can be used by individual sections on a page to place content. It is helpful for highlighting a section by giving it a distinct layout. It includes an introduction, main content area, and an optional link.
+
+<Canvas>
+  <Story name="Default">{defaultDemo}</Story>
+</Canvas>
```

Comment:
@gerardo-rodriguez Could you provide guidance to @dromo77 on how to provide a source code example for this without duplicating her template code?

Reply:
Sure thing! I'll take a closer look this morning. 😉

Reply:
@dromo77 Alrighty, let's look into adding a more helpful source code example. The goal is to arrive at an end result like the following:
<img width="1392" alt="Screen Shot 2021-07-14 at 10 55 16 AM" src="https://user-images.githubusercontent.com/459757/125669814-07727389-fcf8-459c-8175-94a5eb636f9c.png">


We'll follow [a previous PR I merged](https://github.com/cloudfour/cloudfour.com-patterns/pull/1359/files) as an example. 🙂 To set it up, we can leverage the `demo.twig` template you created. This includes:
1. `import` your `demo.twig` file as raw source code
2. Set the imported raw source code as the `docs.source.code` value for the story's `parameters` attribute

Below are the changes you'll need to make. Notice:

- We import raw source code from your template using the non-standard, Webpack-specific`!!raw-loader!` syntax
- We add a helpful comment because it's a non-standard syntax
- We need to disable ESLint for the non-standard syntax `import` line
- We set the `docs.source.code` value to the raw code from your template.

```suggestion
import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
// The '!!raw-loader!' syntax is a non-standard, Webpack-specific, syntax.
// See: https://github.com/webpack-contrib/raw-loader#examples
// For now, it seems likely Storybook is pretty tied to Webpack, therefore, we are
// okay with the following Webpack-specific raw loader syntax. It's better to leave
// the ESLint rule enabled globally, and only thoughtfully disable as needed (e.g.
// within a Storybook docs page and not within an actual component).
// This can be revisited in the future if Storybook no longer relies on Webpack.
// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
import defaultDemoSource from '!!raw-loader!./demo/demo.twig';
import defaultDemo from './demo/demo.twig';

<Meta title="Objects/Feature Group" />

# Feature Group

The Feature Group layout object can be used by individual sections on a page to place content. It is helpful for highlighting a section by giving it a distinct layout. It includes an introduction, main content area, and an optional link.

<Canvas>
  <Story
    name="Default"
    parameters={{
      docs: {
        source: {
          code: defaultDemoSource,
        },
      },
    }}
  >
    {defaultDemo}
  </Story>
</Canvas>
```



Feel free to start with the above suggestion. In theory, it _should_ work but if it doesn't let me know and I can help troubleshoot. I have experienced the MDX files to be a bit touchy so don't hesitate to reach out of something breaks. 😉

Reply:
This is working great, thanks for the helpful explanation @gerardo-rodriguez!